src/pages/workspaces/workspace/analysis/Analyses.ts

```diff
@@ -405,44 +406,61 @@ const Analyses = _.flow(
         div({
           style: { display: 'flex', alignItems: 'center' }
         }, [
-          label({ htmlFor: id, style: { fontWeight: 'bold', margin: '0 0.5rem', whiteSpace: 'nowrap' } }, 'Enable JupyterLab'),
-          h(Switch, {
-            id,
-            checked: enableJupyterLabGCP,
+          label({ htmlFor: id, style: { fontWeight: 'bold', margin: '0 0.5rem', whiteSpace: 'nowrap' } }, ['Enable JupyterLab']),
+          h(Switch as React.FC<any>, {
```

Comment:
If types are an issue with the untyped Switch component, `@ts-expect-error` would be better than a cast. That way, we know to fix this when adding types to Switch.